README.md

```diff
@@ -1,67 +1,69 @@
-# Proposed members' charter
+# Proposed students' charter
 
 Founders & Coders C.I.C. (FAC)  is a not-for-profit company whose main purpose is to provide study and training programmes for the general public and to provide workspace for its members.
 
-FAC was founded on the principal of reciprocity. That is, the company does not charge fees for its study programmes, but expects its members to contribute according to their abilities to the viability and development of the company.
+FAC was founded on the principal of reciprocity. That is, the company does not charge fees for its study programmes, but expects its students to contribute to the viability and development of the company in return.
 
-In particular, members are expected to participate in our study programmes and to support our goal of being financially sustainable, which will ensure that we can continue to offer study programmes and workspace free of charge long into the future.
+In particular, members of each cohort of students are expected to take the lead on facilitating the course for the next cohort, which will ensure that we can continue to offer study programmes and workspace free of charge long into the future. Our goal is to be financialy sustainable, and the company cannot survive without the continued support of our graduates.
 
-All members must abide by the following rules of membership.
+In addition to providing a free education, FAC has been successful in finding permanent or freelance work for all of our graduates. Our primary source of income is recruitment fees from companies who hire our graduates, and so graduates are also required to give FAC the opportunity to approach employers on their behalf if we do not already have a relationship with them.
```

Comment:
Is it accurate to say that permanent or freelance work has been successfully found for ALL graduates by Founders and Coders? Given our recruitment/agency strategy is still a work in progress, perhaps we might want to reword this a little more modestly.

Also, genuine question: is recruitment fees currently our primary source of income? Or is this something we're aiming to establish in the future?